command/meta_backend.go

```diff
@@ -1343,9 +1343,8 @@ func (m *Meta) backend_C_R_S_unchanged(
 	}
 
 	if output {
-		m.Ui.Output(m.Colorize().Color(fmt.Sprintf(
-			"[reset][green]\n\n"+
-				strings.TrimSpace(successBackendLegacyUnset), s.Backend.Type)))
+		m.Ui.Output(m.Colorize().Color(
+			"[reset][green]\n\n" + strings.TrimSpace(successBackendLegacyUnset)))
```

Comment:
I think that having `s.Backend.Type` here for the context would be still helpful. It also means we'll need to change the message (`successBackendLegacyUnset`). How about something like this?

```
Terraform has successfully migrated from legacy remote state to your
configured backend (%q).
```